packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -26,6 +37,86 @@ export class Interpreter {
     this.runtime = <Runtime>{};
   }
 
+  /**
+   * Fetches Asset Definition for given asset index from current context (ctx)
+   * @param assetId Asset Index
+   */
+  getAssetDef (assetId: number): AssetDef | undefined {
+    const accountAddr = this.runtime.ctx.state.assetDefs.get(assetId);
+    if (!accountAddr) return undefined;
+
+    let account = this.runtime.ctx.state.accounts.get(accountAddr);
+    account = this.runtime.assertAccountDefined(account);
+
+    return account.createdAssets.get(assetId);
+  }
+
+  /**
+   * fetches app from `this.store`
+   * @param appId Application Index
+   */
+  getApp (appId: number, line: number): SSCAttributesM | undefined {
```

Comment:
[`SSCAttributesM`](https://github.com/scale-it/algorand-builder/blob/master/packages/runtime/src/types.ts#L86) represents [`SSCAttributes`](https://github.com/scale-it/algorand-builder/blob/master/packages/types-algosdk/index.d.ts#L549) from algosdk, with one stipulation: we are using `maps` instead of `arrays` ( in `'global-state'` here). 
Added the suffix `M` as we are using maps. We can rename to something more meaningful if not clear.